reproducible/debootstrap.bzl

```diff
@@ -99,3 +99,82 @@ def debootstrap_image(name, variant="minbase", distro="jessie", overlay_tar="",
         tars=tars,
         env=env,
     )
+
+def _ubuntu_impl(ctx):
+    # Strip off the '.tar'
+    image_name = ctx.attr._builder_image.label.name.split('.', 1)[0]
+    # docker_build rules always generate an image named 'bazel/$package:$name'.
+    builder_image_name = "bazel/%s:%s" % (ctx.attr._builder_image.label.package,
+                                          image_name)
+
+    # Generate a shell script to run the build.
+    build_contents = """\
+#!/bin/bash
+set -ex
+
+# Download the official ubuntu .tar.gz
+# TODO(aaron-prindle) remove builder-image dep/input and variable from ubuntu script
+# {1} = builder_image_name (bazel/reproducible:builder)
+# {2} = variant (base)
+# {3} = release (16.04)
+# {4} = .tar.gz name/path (bazel-out/local-fastbuild/bin/reproducible/ubuntu.rootfs.tar.gz)
+curl -o {4} http://cdimage.ubuntu.com/ubuntu-{2}/releases/{3}/release/ubuntu-{2}-{3}-core-amd64.tar.gz
```

Comment:
Hmm, I think this might be a bit overcomplicated for now. You might be able to getaway with doing something like this:

- add the .tar.gz in your WORKSPACE file, via http_file: https://docs.bazel.build/versions/master/be/workspace.html#http_file
- use a basic docker_image rule in the BUILD file to create a single-layer image.

It would be nicer later once we slap on some configs to keep the ubuntu_image helper as a macro, but for now you should be fine with just http_file and docker_image.